### PR TITLE
Replace buefy components with csc-ui components

### PR DESCRIPTION
**Pull request by csc-felipe**
### Description

Many of buefy components used could already be changed to use `csc-ui` components.

### Changes Made

- Replace buefy b-input with c-text-field from `csc-ui`
- Replace buefy b-button with c-button from `csc-ui`
- Replace buefy b-select with c-select from `csc-ui`
- Replace buefy b-loading with c-loader and remove unused b-loading
- Replace buefy b-table with c-data-table from `csc-ui`

### Testing

- [x] Tests do not apply
- [ ] Manually test changed functionality. Changed components
  - [ ] `CopyFolderModal.vue` - input fields
  - [ ] `CreateFolderModal.vue` - input fields
  - [ ] `ObjectTable.vue` - search box
  - [ ] `SelectPage.vue` - text input, dropdown, buttons
  - [ ] `Tokens.vue` - text input, dropdown, buttons, data table

### Mentions
As pointed out by @blankdots, buefy text fields look quite different from csc-ui, as csc-ui text fields look like the ones in material design from Google.

cc @hannyle, @ainoc 

- csc-ui
image
- buefy
image

### CODE REVIEW

File: swift_browser_ui_frontend/src/components/CopyFolderModal.vue [hunk 1]
```diff
@@ -15,23 +15,18 @@
...
-            v-model="folderName"
-            name="foldername"
-            custom-class="has-text-dark"
-            :loading="loadingFoldername"
-          />
-        </b-field>
+          name="foldername"
+          :loading="loadingFoldername"
+        />
         <label
```
Comment (blankdots) [inline swift_browser_ui_frontend/src/components/CopyFolderModal.vue:26]: wondering if we should keep the label, i see that CSC design system is not using labels to text fields at all, wondering if we should scrape this in favour of `label` attribute inside `c-text-field` ?

Comment (csc-felipe) [reply swift_browser_ui_frontend/src/components/CopyFolderModal.vue:26]: I've been wondering that as well. They seem to use the same concept from Google's material design.

cc @hannyle, @ainoc

Comment (hannyle) [reply swift_browser_ui_frontend/src/components/CopyFolderModal.vue:26]: Not sure I follow. 
`c-text-field` has `label` attribute. Which `label` are you referring to?

Comment (hannyle) [reply swift_browser_ui_frontend/src/components/CopyFolderModal.vue:26]: Ahh I understand what you mean after reading the Description again.

Comment (hannyle) [reply swift_browser_ui_frontend/src/components/CopyFolderModal.vue:26]: I personally think the `label` together with `hint` in `c-text-field` is clear enough for user.

**Comment by csc-felipe** [comment]
rebased

File: swift_browser_ui_frontend/src/components/UploadModal.vue [hunk 2]
```diff
@@ -115,7 +115,7 @@
                   rows="3"
                 />
                 <c-button
-                  type="is-success"
+                  type="success"
```
Comment (hannyle) [inline swift_browser_ui_frontend/src/components/UploadModal.vue:118]: I think there is no type of `success` for `c-button`, only type `button` or `submit`

File: swift_browser_ui_frontend/src/components/UploadModal.vue [hunk 3]
```diff
@@ -115,7 +115,7 @@
...
                   icon-left="lock-plus"
```
Comment (hannyle) [inline swift_browser_ui_frontend/src/components/UploadModal.vue:119]: I think this `icon-left` attribute is from Buefy.

File: swift_browser_ui_frontend/src/views/Tokens.vue [hunk 4]
```diff
@@ -54,70 +36,29 @@
         <span>
           <b>{{ $t('message.tokens.latestToken') }}</b>&nbsp;{{ latest }}&nbsp;
         </span>
-        <b-button
+        <c-button
           class="copyButton"
           outlined
           icon-left="content-copy"
           @click="copyTokenHex()"
```
Comment (hannyle) [inline swift_browser_ui_frontend/src/views/Tokens.vue:43]: I think this `icon-left` attribute is from Buefy.

File: swift_browser_ui_frontend/src/components/UploadModal.vue (hunk 2 above)
Comment (csc-felipe) [reply swift_browser_ui_frontend/src/components/UploadModal.vue:118]: yes, there's no way to have other colors in the buttons. Is it ok to use default button here?

Comment (hannyle) [reply swift_browser_ui_frontend/src/components/UploadModal.vue:118]: Yes, I think the default - primary color is good enough

**Comment by csc-felipe** [comment]
resolved comments

**Review by blankdots — APPROVED** [review]

File: swift_browser_ui_frontend/src/components/ObjectTable.vue [hunk 5]
```diff
@@ -79,15 +79,19 @@
...
-          v-model="searchQuery"
-          :placeholder="$t('message.objects.filterBy')"
-          type="search"
-          icon="filter-variant"
+      <c-text-field
+        id="search"
+        v-csc-model="searchQuery"
+        name="search"
+        custom-class="has-text-dark"
+        :placeholder="$t('message.objects.filterBy')"
```
Comment (hannyle) [inline swift_browser_ui_frontend/src/components/ObjectTable.vue:87]: This `custom-class` is also not part of `c-text-field`

Comment (csc-felipe) [reply swift_browser_ui_frontend/src/components/ObjectTable.vue]: removed

**Comment by csc-felipe** [comment]
Removed `custom-class` from `c-text-field`

**Review by hannyle — APPROVED** [review]

**Review by blankdots — APPROVED** [review]

**Pull request merged**